### PR TITLE
Better crystal aura

### PR DESCRIPTION
**Pull request by N0tExisting**
Improved crystal aura Autoplace & Destruction

### CODE REVIEW

File: Horion/Module/Modules/CrystalAura.cpp [hunk 1]
```diff
@@ -53,13 +54,24 @@ bool CanPlaceC(vec3_ti* pos) {
 	g_Data.forEachEntity([](C_Entity* ent, bool b) {
 		if (!space)
 			return;
-		if (ent->aabb.intersects(*new AABB(_pos, _pos.add(1.f))))
+		if (ent->aabb.intersects(*new AABB(_pos, 1.f, 1.f)))
```
Comment (ghost) [inline Horion/Module/Modules/CrystalAura.cpp]: AABB::AABB': no overloaded function takes 3 arguments

Comment (ghost) [reply Horion/Module/Modules/CrystalAura.cpp]: it can handle 4 tho?????

Comment (N0tExisting) [reply Horion/Module/Modules/CrystalAura.cpp]: I made a custom overload then realized that u can just use add, but I was commiting to 2 different branches so I probably overwrote it.
Gonna fix it later

Comment (N0tExisting) [reply Horion/Module/Modules/CrystalAura.cpp]: Fixed it

**Review by Drosh23 — APPROVED** [review]

**Comment by GodGamer029** [comment]
This code is terrible ngl, still merging

**Pull request merged**